.github/workflows/release.yml

```diff
@@ -84,20 +86,24 @@ jobs:
               generate_release_notes: true
             })
 
-      - name: Check commit messages for feat or break
+      - name: Check commit messages and descriptions for feat or break
         id: check_commits
         run: |
           PUBLISH_DOCS=false
+          COMMIT_DETAILS=""
           for COMMIT in $(git rev-list ${{ github.event.before }}..${{ github.sha }}); do
             COMMIT_MESSAGE=$(git log -1 --pretty=%B $COMMIT)
+            COMMIT_DESCRIPTION=$(git log -1 --pretty=%b $COMMIT)
             echo "Commit message: $COMMIT_MESSAGE"
+            echo "Commit description: $COMMIT_DESCRIPTION"
+            COMMIT_DETAILS="$COMMIT_DETAILS\n- $COMMIT_MESSAGE: $COMMIT_DESCRIPTION"
             if echo "$COMMIT_MESSAGE" | grep -E "feat:|break:"; then
               PUBLISH_DOCS=true
-              break
             fi
           done
           echo "Publish docs to Confluence: $PUBLISH_DOCS"
```

Comment:
The if condition is checking for "feat:" or "break:" in the commit message, but it's missing a colon after "break" which could lead to incorrect detection.